low-level/framework/base/global_config.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/python3.6
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+from cortx.utils.conf_store import Conf
+from framework.utils.conf_utils import (GLOBAL_CONF, SSPL_CONF,
+            CLUSTER, SERVER_NODES, MACHINE_ID, STORAGE, STORAGE_SET_ID,
+            ENCLOSURE_ID, CONTROLLER, PRIMARY, IP, PORT, SECONDARY,
+            USER, PRODUCT, RSYSLOG, HOST, NODE_TYPE, TYPE, RELEASE,
+            TARGET_BUILD, SECRET, CONF_API_DELIMITER)
+from framework.base.sspl_constants import (DATACENTER01, RACK01, SERVERNODE01,
+                CLUSTER01, DEFAULT_MC_IP, DEFAULT_SYSLOG_HOST, DEFAULT_SYSLOG_PORT)
+
+
+class GlobalConf:
+    """Base Class to fetch global config values."""
+
+    Conf.load(SSPL_CONF, "yaml:///etc/sspl.conf")
```

Comment:
why this code(line 31-33) here? can stay in __init__

Reply:
For each sspl service process, we can't load the Conf more than once.
If we keep this block of code in init, whenever each sensor will create an obj, 
 Conf.load(GLOBAL_CONF, global_config)
 will be called, which raises the error.